com.unity.ml-agents/Runtime/Analytics/InferenceAnalytics.cs

```diff
@@ -1,17 +1,31 @@
+#if MLA_UNITY_ANALYTICS_MODULE || !UNITY_2019_4_OR_NEWER
+#define MLA_UNITY_ANALYTICS_MODULE_ENABLED
+#endif
+
+#if MLA_UNITY_ANALYTICS_MODULE_ENABLED
+#endif
```

Comment:
```suggestion
```
Redundant lines? Or did Rider auto-refactor something?

Reply:
remnants of old debugging code. thanks for catching it